Lesson 6

GriKate opened the pull request:
д\з 6

GriKate committed:
Merge branch 'lesson-5'

GriKate committed:
д\з 6

ubcent commented on .idea/js-geekbrains.iml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Нужно добавить все эти файлы в игнор, они не должны присутствовать в репозитории

ubcent commented on homework-6.js:
```diff
@@ -0,0 +1,73 @@
+    if (products.length > 0) {
+        products.push(productObj);
+        for (var i = 0; i < products.length - 1; i++) {
+            if (productObj.name === products[i].name && productObj.price === products[i].price) {
```
Не очень правильно ориентироваться на название и цену, лучше добавить какое-нибудь уникальное поле. Обычно оно называется id

ubcent commented on homework-6.js:
```diff
@@ -0,0 +1,73 @@
+    $productButton.dataset.price = productPrice;
+    productObj.price = +$productButton.dataset.price;
+
+    // productObj.quantity++;
```
Не стоит оставлять закомментированный код в чистовом решении

GriKate committed:
переделала д/з.
Написана генерация страницы каталога, генерация корзины (отображается при наведении) и открытие картинки товара в модальном окне
